Acceleratio.XCellKit/SpredsheetChart.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using DocumentFormat.OpenXml.Drawing.Charts;
+using DocumentFormat.OpenXml.Packaging;
+
+namespace Acceleratio.XCellKit
+{
+    public class SpredsheetChart
+    {
+        public ChartTypeEnum ChartType { get; private set; }
+        public List<ChartModel> ChartData { get; private set; }
+        public List<GanttData> GanttData { get; private set; }
+        public ChartSettings Settings { get; set; }
+
+        protected SpredsheetChart(SpredsheetChart export)
+        {
+            this.ChartType = export.ChartType;
+            this.ChartData = export.ChartData;
+            this.GanttData = export.GanttData;
+            this.Settings = export.Settings;
+        }
+
+        public SpredsheetChart(ChartTypeEnum chartType, List<ChartModel> chartData, ChartSettings settings)
+        {
+            this.ChartType = chartType != ChartTypeEnum.Gantt ? chartType : ChartTypeEnum.Bar;
```

Comment:
this means that you cannot use this constructor if the chartTypes is gantt?
If that's the case maybe throw an Exception?